packages/react-form/src/utilities.ts

```diff
@@ -40,4 +49,51 @@ export function isChangeEvent(
   );
 }
 
+export function propagateErrors(
+  fieldBag: {[key: string]: FieldOutput<any>},
+  remoteErrors: FormError[],
```

Comment:
nit pick, it is named `remoteErrors` here, but we also pass in `clientError` above?

Reply:
It should really just be called `errors` here, as it gets used both for errors from running all the local validations and for errors from the submit callback.